classes/basement/rootrecipe.yaml

```diff
@@ -16,6 +16,9 @@ depends:
           - name: devel::bootstrap-fake-pkg-config
             use: [tools]
             forward: True
+          - name: devel::bootstrap-fake-autotools
```

Comment:
Is this really necessary? Apparently `devel::m4` does not need it. If I disable `devel::bootstrap-fake-autotools`, I don't get any parsing error either. Do I overlook something? :thinking:

Reply:
Hmm...looks like this is no longer necessary... These commits are ~4years old I did not clean them up carefully enough...sorry for that...